src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -256,6 +256,12 @@ pub fn print_version() {
     }
 }
 
+pub fn get_version() -> (u32, u32) {
+    let major = unsafe { llvm::LLVMRustVersionMajor() };
```

Comment:
any other calls in the compiler to this function that we could unify under `get_version`?

Reply:
I've replaced two more occurrences. I've also changed this to only provide the major version, as LLVM no longer uses the minor.

At this point the function provides no benefit apart from saving an unsafe, maybe I should drop it altogether?

Reply:
I like the safe wrapper. The use sites are definitely prettier this way.